lib/src/message.dart

```diff
@@ -4,4 +4,26 @@ import 'package:nyxx_extensions/src/channel.dart';
 extension MessageUtils on Message {
   /// A URL clients can visit to navigate to this message.
   Future<Uri> get url async => Uri.https(manager.client.apiOptions.host, '${(await channel.get()).url.path}/$id');
+
+  /**
+   * Sends a reply to the message,
+   */
+  Future<Message> sendReply(MessageBuilder builder) {
+    final copiedBuilder = MessageBuilder(
+        allowedMentions: builder.allowedMentions,
+        attachments: builder.attachments,
+        components: builder.components,
+        content: builder.content,
+        embeds: builder.embeds,
+        nonce: builder.nonce,
+        replyId: this.id,
+        requireReplyToExist: builder.requireReplyToExist,
+        stickerIds: builder.stickerIds,
+        suppressEmbeds: builder.suppressEmbeds,
+        suppressNotifications: builder.suppressNotifications,
+        tts: builder.tts,
+    );
```

Comment:
I think you dont need to copu message builder, just replace replyId in builder from parameter

Reply:
Also would add info to info to method docs that it modifies passed builder with new replyId value

Reply:
Just in case if someone wanted to like reuse the same message builder that they used with `sendReply()`. Copying it will allow the developer to reuse that builder again. Do you still think we should directly modify the original builder?

Reply:
ah yes, that makes sense. Okay then

Reply:
Alright!